### PR TITLE
gmp/zarith: build on freebsd

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

File: packages/zarith/zarith.1.3/opam [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+  ["./configure"] { os != "openbsd" & os != "freebsd" }
+  ["env" "CFLAGS=-I/usr/local/include" "LDFLAGS=-L/usr/local/lib" "./configure"] { os = "openbsd" | os = "freebsd" }
   [make]
-  [make "install"]
 ]
-remove: [["ocamlfind" "remove" "zarith"]]
+install: [make "install"]
+remove:  ["ocamlfind" "remove" "zarith"]
 depends: [
+  "ocamlfind" {build}
```
Comment (avsm) [inline packages/zarith/zarith.1.3/opam]: I think using `{build}` requires opam-version to be 1.2

**Comment by seliopou** [comment]
Might need a little guidance on this one. Not sure what changes will satisfy the least-common denominator opam version of `1.1.2`.

File: packages/zarith/zarith.1.3/opam [hunk 2]
```diff
@@ -1,13 +1,15 @@
 opam-version: "1"
 maintainer: "contact@ocamlpro.com"
+authors: "OCamlPro"
```
Comment (avsm) [inline packages/zarith/zarith.1.3/opam]: I don't think the authors of this are OCaml Pro -- I think it's Xavier Leroy

**Comment by samoht** [comment]
The 1.1.2 convert scripts needs updating ... /cc @AltGr

**Pull request merged**